README.md

```diff
@@ -1,30 +1,32 @@
-### Quick note
-I wrote this as a quick evening script to solve my problem. If you're looking for something more robust & need more than this script can give you, check out:
-https://github.com/sdias/win-10-virtual-desktop-enhancer
-
 # windows-desktop-switcher
-An AutoHotKey script for Windows that lets a user change virtual desktops by pressing CapsLock + &lt;num>. It also allows for creation/deletion of desktops by hotkey (see below).
+An AutoHotKey script for Windows that lets a user change virtual desktops by pressing CapsLock + &lt;num> or other custom key combination. It also provides other features, such as creation/deletion of desktops by hotkey, etc. (see Hotkeys section below).
 
 ## Overview
-This script creates 'better' hotkeys for switching virtual desktops in windows 10. I built this to better mirror
+This script creates more convenient hotkeys for switching virtual desktops in Windows 10. I built this to better mirror
 the mapping I use on linux (with dwm), and it's always annoyed me that Windows does not have better
 hotkey support for this feature (for instance, there's no way to go directly to a desktop by number).
 
-Note that this only overrides CapsLock for the key combinations below. Otherwise, CapsLock will function normally.
+Note, CapsLock will function normally even when using it as a modifier.
 
 ## Installation
-Install AutoHotKey v1.1 or later, then run the desktop_switcher.ahk script (open with AutoHotKey if prompted). I would recommend putting it in your startup folder and it'll be invoked on login.
+[Install AutoHotkey](https://autohotkey.com/download/) v1.1 or later, then run the desktop_switcher.ahk script (open with AutoHotKey if prompted). I would recommend putting it in your startup folder and it'll be invoked on login. 
+
+Note, on some windows (such as the ones you've run with Administrator privileges, also terminals) the hotkeys will only work if the script itself is `Run as Administrator`, due to the way Windows is designed. For the script to be invoked at startup with Administrator privileges, you can use Windows Task Scheduler.
```

Comment:
I put a shortcut to this script in the shell:startup folder and it appears to successfully run as admin. I think this may be an easier method - we could consider adding this script in if users need an easy way to bootstrap a run-as-admin setup. Alternatively we could just document the method (or both). 

desktop_switcher_admin.vbs
```
Set WshShell = CreateObject("WScript.Shell" ) 
WshShell.Run """desktop_switcher.ahk""", 0 
Set WshShell = Nothing
```

Reply:
I think documenting is enough, no need to make the repo heavier.

For now even mentioning that it is possible would be enough :)